bedrock/newsletter/views.py

```diff
@@ -36,163 +35,140 @@
 
 log = commonware.log.getLogger('b.newsletter')
 
-LANG_FILES = ['mozorg/newsletters']
-general_error = _lazy(u'We are sorry, but there was a problem '
-                      u'with our system. Please try again later!')
-thank_you_new = _lazy(u'Your email preferences have been successfully updated.')
-thank_you = _lazy(u'Thanks for updating your email preferences.')
-bad_token = _lazy(u'The supplied link has expired or is not valid. You will '
-                  u'receive a new one in the next newsletter, or below you '
-                  u'can request an email with the link.')
-recovery_text = _lazy(
-    u'Success! An email has been sent to you with your preference center '
-    u'link. Thanks!')
-
-# NOTE: Must format a link into this: (https://www.mozilla.org/newsletter/)
-unknown_address_text = _lazy(
-    u'This email address is not in our system. Please double check your '
-    u'address or <a href="%s">subscribe to our newsletters.</a>')
-
-invalid_email_address = _lazy(u'This is not a valid email address. '
-                              u'Please check the spelling.')
+FTL_FILES = ['mozorg/newsletters']
+
+general_error = ftl_lazy('newsletters-we-are-sorry-but-there', ftl_files=FTL_FILES)
```

Comment:
Not sure if there is a cleaner way than adding `ftl_files=FTL_FILES` to each of these `ftl_lazy()` calls?

Reply:
Unfortunately not at the moment. If `newsletters` is going to be a default file then we could omit the `ftl_files` param though.